openscm/regions.py

```diff
@@ -2,7 +2,7 @@
 Handling of region information.
 """
 
-from typing import Dict, Tuple
+from typing import cast, Dict, Optional, Tuple
 from .errors import RegionAggregatedError
 from . import parameters  # pylint: disable=cyclic-import
```

Comment:
why isn't the cyclic-import a problem?

Reply:
Cyclic imports are not necessarily a problem in Python (because of the `sys.modules` caching). This import, however, is necessary for Sphinx to find the correct types. Would be happy to do that in a better way, but have not found any.

Reply:
ah ok fair